nike_sneaker_store/lib/components/cards/card_category.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:nike_sneaker_store/resources/ns_style.dart';
+
+class CardCategory extends StatelessWidget {
+  const CardCategory({
+    required this.text,
+    this.onPressed,
+    this.backgroundColor,
+    this.textColor,
+    super.key,
+  });
+
+  final String text;
+  final Function()? onPressed;
+  final Color? backgroundColor;
+  final Color? textColor;
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: onPressed,
+      child: Container(
+        padding: const EdgeInsets.symmetric(horizontal: 23, vertical: 11),
+        decoration: BoxDecoration(
+          color:
+              backgroundColor ?? Theme.of(context).colorScheme.primaryContainer,
+          borderRadius: BorderRadius.circular(8),
+        ),
+        child: Text(
+          text,
+          style: NSStyle.h12Normal.copyWith(
```

Comment:
@huy2992002 : You should check all. All style should get from theme, when change any style in theme, it will change all place.